com.unity.ml-agents/Runtime/Analytics/TrainingAnalytics.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Collections.Generic;
+using Unity.Barracuda;
+using Unity.MLAgents.Actuators;
+using Unity.MLAgents.Inference;
+using Unity.MLAgents.Policies;
+using Unity.MLAgents.Sensors;
+using UnityEngine;
+using UnityEngine.Analytics;
+
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditor.Analytics;
+#endif
+
+namespace Unity.MLAgents.Analytics
+{
+    internal class TrainingAnalytics
+    {
+        const string k_VendorKey = "unity.ml-agents";
+        const string k_TrainingEnvironmentInitializedEventName = "ml_agents_trainingenvironmentinitialized";
```

Comment:
Need better names.

Reply:
We should call a cat a cat. ml_agents_training_environment_initialized tells it like it is. It is an event from the environment, that is sent when the training is initialized.

In that spirit, I would call ml_agents_behaviortrainingstarted -> ml_agents_training_behavior_initialized
and ml_agents_remotepolicystarted -> ml_agents_training_policy_initialized (since a policy that is training has to be remote in any way)